prisma/migrations/20250514145741_unique_device_identity_on_user_id_and_xmtp_id/migration.sql

```diff
@@ -0,0 +1,35 @@
+/* Query to find DeviceIdentity records that should be deleted */
+DELETE FROM "DeviceIdentity"
```

Comment:
This looks good to me. I did a little investigating in Retool. There are 102 total rows with dupes. This will delete 86 of them, across 16 different userID/xmtpID pairings.

`102-16 == 86`, so I think we're good!